drive/drive.py

```diff
@@ -212,9 +214,11 @@ def add_options(self, chromiumlike, cache_state):
             driver_options.add_argument('--sparrow-force-fieldtrial')
             driver_options.add_argument('--user-data-dir=%s' % self.sparrow_user_data_dir)
             for switch in self.sparrow_only_switches:
+                if self.fresh_models and switch.split('=')[0].strip() == '--hinting-scope':
+                    switch += '-%s' % time.time()
```

Comment:
time.time() returns decimal point which turns into a period character '.'.  Just want to make sure it is supported as testLabel

Reply:
thank you for pointing that out